portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="content">
+      <h1>Projects</h1>
+      <div id="proj-container">
+        <div class="default-slide">
+          <div class="default-slide-block">
+            Click the forward and backward arrows to view my projects.
+          </div>
+        </div>
+        <div class="slide">
+          <div class="slide-container">
+            <a href="https://www.stanforddaily.com/2019/04/19/modeling-the-draw/">
+              <img src="/images/modeling-the-draw.jpg" alt="Modeling the Draw">
```

Comment:
Because the content of the project slides is already conveyed by the caption text, I think the alt text for the images can be removed. If you do so, be sure to set `alt=""` because I believe that in the absence of any `alt` attribute, many screen readers will read out the full image source URL as a fallback.